unlock-app/src/components/interface/checkout/main/Payment.tsx

```diff
@@ -110,6 +110,13 @@ export function Payment({ injectedProvider, checkoutService }: Props) {
     (item) => !item
   )
 
+  // Universal card is enabled if credit card is not enabled by the lock manager and the lock is USDC
+  const USDC = networkConfig?.tokens?.find((t: any) => t.symbol === 'USDC')
+  const universalCardEnabled =
+    !enableCreditCard &&
+    networkConfig.universalCard?.cardPurchaserAddress &&
+    lock.currencyContractAddress === USDC.address
```

Comment:
```suggestion
    lock.currencyContractAddress === USDC?.address
```

Reply:
USDC and currencyContractAddress can be both undefined or in different case (not normalized).